### PR TITLE
Automated cherry pick of #16178

### PR DESCRIPTION
**Pull request by madhusudancs**
Cherry pick of #16178 on release-1.1.

### CODE REVIEW

**Comment by madhusudancs** [comment]
@davidopp @dchen1107 PTAL.

**Comment by dchen1107** [comment]
LGTM Thanks for the fix.

**Comment by davidopp** [comment]
LGTM

**Comment by dchen1107** [comment]
ok to test

File: pkg/kubelet/kubelet_test.go [hunk 1]
```diff
@@ -2496,6 +2496,23 @@ func TestUpdateNewNodeStatus(t *testing.T) {
 		DockerVersion:      "1.5.0",
 	}
 	mockCadvisor.On("VersionInfo").Return(versionInfo, nil)
+
+	// Create a new DiskSpaceManager with a new policy. This new manager along with the mock
+	// FsInfo values added to Cadvisor should make the kubelet report that it has sufficient
+	// disk space.
+	dockerimagesFsInfo := cadvisorapiv2.FsInfo{Capacity: 500 * mb, Available: 200 * mb}
```
Comment (dchen1107) [inline pkg/kubelet/kubelet_test.go]: You need import:

cadvisorapiv2 "github.com/google/cadvisor/info/v2"

**Comment by dchen1107** [comment]
@madhusudancs please resolve above cherry-pick issue. Thanks!

File: pkg/kubelet/kubelet_test.go (hunk 1 above)
Comment (dchen1107) [inline pkg/kubelet/kubelet_test.go]: Ahh, the package alias is cadvisorApiv2 in 1.1 branch, not cadvisorapiv2 :-(

Comment (dchen1107) [inline pkg/kubelet/kubelet_test.go]: https://github.com/kubernetes/kubernetes/issues/15319

**Comment by madhusudancs** [comment]
@dchen1107 PTAL.

**Comment by dchen1107** [comment]
LGTM

**Pull request merged**